configuration/configuration-mainnet.yaml

```diff
@@ -26,15 +26,15 @@ defaultBackends:
 
 # here we set up outputs of logging in 'katip':
 setupScribes:
-  - scKind: StdoutSK
-    scName: stdout
+  - scKind: FileSK
+    scName: "logs/mainnet.log"
     scFormat: ScText
-    scRotation: null
 
 # if not indicated otherwise, then log output is directed to this:
 defaultScribes:
-  - - StdoutSK
-    - stdout
+  - - FileSK
+    - "logs/mainnet.log"
+    #  - - DevNullSK
```

Comment:
Commented out

Reply:
clearly a leftover that should have been removed.